code/game/gamemodes/clock_cult/clock_items/clockwork_slab.dm

```diff
@@ -1,8 +1,8 @@
 /obj/item/clockwork/slab //Clockwork slab: The most important tool in Ratvar's arsenal. Allows scripture recital, tutorials, and generates components.
 	name = "clockwork slab"
 	desc = "A strange metal tablet. A clock in the center turns around and around."
-	clockwork_desc = "A link between the Celestial Derelict and the mortal plane. Contains limitless knowledge, fabricates components, and outputs a stream of information that only a trained eye can detect.\n\
-	Use the <span class='brass'>Hierophant Network</span> action button to communicate with other servants.\n\
+	clockwork_desc = "A link between you and the City of Cogs. It produces components,  contains information, and is your most vital tool as a servant.\n\
```

Comment:
I do prefer Celestial Derelict.
Also capitalize servant here and on the line below